src/en/components/button/base.md

```diff
@@ -15,10 +16,15 @@ A button is an interactive object that highlights an important or common action.
 {% docLinks locale stage figma github %}
 {% enddocLinks %}
 
-<div class="b-sm b-gray px-250 py-400 my-500">
-  <gcds-button button-role="primary">Primary label</gcds-button>
-  <gcds-button button-role="secondary">Secondary label</gcds-button>
-  <gcds-button button-role="danger">Danger label</gcds-button>
-  <gcds-button button-role="skip-to-content">Skip-to-content label</gcds-button>
-  <gcds-button type="link" button-style="text-only" href="#">Text-only label</gcds-button>
+<div class="my-500 b-sm b-gray">
```

Comment:
Can we able to make this component preview box a paired shortcode? That way we only have to modify the style in one place if we want to change it